src/flutter/shell/platform/linux_embedded/surface/context_egl.h

```diff
@@ -23,6 +27,24 @@ class ContextEgl {
   ContextEgl(std::unique_ptr<EnvironmentEgl<D>> environment)
       : environment_(std::move(environment)), config_(nullptr) {
     EGLint config_count = 0;
+#if defined(DISPLAY_BACKEND_TYPE_EGLSTREAM)
+    const EGLint attribs[] = {EGL_SURFACE_TYPE,
```

Comment:
Please use the original format of `attribs`.

Reply:
I fixed it.